modules/administration-guide/examples/system-variables.adoc

```diff
@@ -658,7 +658,7 @@ Default::: `+<username>-che+`
 
 == `+CHE_INFRA_KUBERNETES_NAMESPACE_LABEL+`
 
-Defines whether che-server should try to label the workspace namespaces.
+Defines whether che-server should try to label the workspace namespaces. NOTE: It is strongly recommended to keep the value of this property set to true. If false, the new workspace namespaces will not be labeled automatically and therefore not recognized by the {prod-short} operator making some features of DevWorkspaces not working. If false, an administrator is required to label the namespaces manually using the labels specified in che.infra.kubernetes.namespace.labels. If you want to manage the namespaces yourself, make sure to follow \https://www.eclipse.org/che/docs/stable/administration-guide/provisioning-namespaces-in-advance/. Any additional labels present on the namespace are kept in place and do not affect the functionality. Also note that the the administrator is free to pre-create and label the namespaces manually even if this property is true. No updates to the namespaces are done if they already conform to the labeling requirements.
```

Comment:
This section could use some rewording (e.g. two consecutive sentences start with `If false,`). Likely the absolute docs link should be replaced with a relative, in-repo link for downstreaming.

Also it's not clear to me the relation between `CHE_INFRA_KUBERNETES_NAMESPACE_LABEL` and  `che.infra.kubernetes.namespace.labels`:
> If false, an administrator is required to label the namespaces manually using the labels specified in che.infra.kubernetes.namespace.labels

```suggestion
Defines whether che-server should try to label the workspace namespaces. NOTE: It is strongly recommended to keep the value of this property set to true. If false, the new workspace namespaces will not be labeled automatically and therefore not recognized by the {prod-short} operator making some features of DevWorkspaces not work. If false, an administrator is required to label the namespaces manually using the labels specified in `che.infra.kubernetes.namespace.labels`. If you want to manage the namespaces yourself, make sure to follow [relative-link]. Any additional labels present on the namespace are kept in place and do not affect the functionality. Also note that the the administrator is free to pre-create and label the namespaces manually even if this property is true. No updates to the namespaces are done if they already conform to the labeling requirements.
```